tests/regression/test_expressions.py

```diff
@@ -8,44 +8,51 @@
 import ufl  # noqa: used in eval'd expressions
 
 from firedrake import *
-from tests.common import *
 
 
-@pytest.fixture(scope='module', params=['cg1', 'cg1cg1[0]', 'cg1cg1[1]',
-                                        'cg1vcg1[0]', 'cg1dg0[0]', 'cg1dg0[1]',
-                                        'cg2dg1[0]', 'cg2dg1[1]'])
-def sfs(request, cg1, cg1cg1, cg1vcg1, cg1dg0, cg2dg1):
-    """A parametrized fixture for scalar function spaces."""
-    return {'cg1': cg1,
-            'cg1cg1[0]': cg1cg1[0],
-            'cg1cg1[1]': cg1cg1[1],
-            'cg1vcg1[0]': cg1vcg1[0],
-            'cg1dg0[0]': cg1dg0[0],
-            'cg1dg0[1]': cg1dg0[1],
-            'cg2dg1[0]': cg2dg1[0],
-            'cg2dg1[1]': cg2dg1[1]}[request.param]
+@pytest.fixture(scope='module')
+def mesh():
+    return UnitSquareMesh(5, 5)
+
+
+@pytest.fixture(scope='module')
+def cg1(mesh):
+    return FunctionSpace(mesh, "CG", 1)
+
+
+@pytest.fixture(scope='module')
+def cg2(mesh):
+    return FunctionSpace(mesh, "CG", 2)
```

Comment:
I've dialled back some of the combinations in this set of tests, because AFAICT they didn't actually increase coverage.